### PR TITLE
✨ Bump k8s and etcd to generate new tools/envtest versions

### PR DESCRIPTION
**Pull request by chrischdi**
**Description:**

* Bump k8s version from v1.28.3 to v1.29.0
* Update ETCD from v3.5.10 to v3.5.11

### CODE REVIEW

**Comment by chrischdi** [comment]
/assign camilamacedo86

:-)

File: build/cloudbuild_tools.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 #  limitations under the License.
 
 substitutions:
-  _KUBERNETES_VERSION: 1.28.3
+  _KUBERNETES_VERSION: 1.29.0
```
Comment (camilamacedo86) [inline build/cloudbuild_tools.yaml:16]: we cannot bump the k8s until controller-runtime and controller-tools have an version using k8s 1.29 released

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

**Review by camilamacedo86 — COMMENTED** [review]
/approved 

/lgtm 

Thank you for the contribution 🥇

**Comment by damdo** [comment]
Thanks for this @chrischdi 

@camilamacedo86 
Would it be possbile/how hard would it be to auto-create pull requests that bump this value, ready to be approved. Or even better, that auto-create and auto-merge? Thanks!

**Comment by camilamacedo86** [comment]
Hi @damdo 

Auto-approved is not safe. So, I think we cannot do that.
However, it might be possible to create a bump with Dependabot. I am not sure. 

Feel free to propose solutions to auto-create the PR

**Pull request merged**